Src/Framework/Jobs/Converter.py

```diff
@@ -13,10 +13,54 @@
 #
 # **************************************************************************
 
+import abc
+
+from Scientific.IO.NetCDF import NetCDFFile
+
 from MDANSE.Framework.Jobs.IJob import IJob
 
 class Converter(IJob):
+
+    __metaclass__ = abc.ABCMeta
     
     category = ('Converters',)
     
     ancestor = ['empty_data']
+
+    @abc.abstractmethod
+    def run_step(self,index):
+        pass
+
+    def finalize(self):
+
+        if not hasattr(self,'_trajectory'):
+            return
+
+        f = NetCDFFile(self._trajectory.filename,'a')
+
+        if 'time' in f.variables:
+            f.variables['time'].units = 'ps'
+            f.variables['time'].axis = 'time'
+            f.variables['time'].name = 'time'
+
+        if 'box_size' in f.variables:
+            f.variables['box_size'].units = 'ps'
+            f.variables['box_size'].axis = 'time'
+            f.variables['box_size'].name = 'box_size'
+
+        if 'configuration' in f.variables:
+            f.variables['configuration'].units = 'nm'
+            f.variables['configuration'].axis = 'time'
```

Comment:
Shouldn't the axis here consist of three variables, given that the data is 3D (i.e. similar to how F(Q, t) has axis of 'q, time')? As it stands, when I take a trajectory that was converted to MMTK NetCDF format using this branch and plot it using the 2D Slice plotter from #61, I get the following, which does have a variable on the x-axis, but does not have one on the y-axis.
![image](https://user-images.githubusercontent.com/78977041/164677959-768bc987-8ada-4572-bf61-c59df4384161.png)

Reply:
Yeap but the time dimension (the first one) is the only dimension that has a sense. The other (number of atoms, and dimension) does not exist. They have no meaning.

Reply:
If the other two dimensions have no meaning, does this 2D slice have any meaning either?

Reply:
That's a good philosophical question. For inspecting trajectories, I would say that the dedicated plugin I developed in branch 58 is much better. However, the 2D Slice, as such, is another way of inspecting 3D data and I would keep it.